### PR TITLE
rubiconBidAdapter_spec.js is using describe.only, stopping other tests from running

### PR DESCRIPTION
**Pull request by msm0504**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Remove duplicate param to fix unit tests

Fix for #4449

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/rubiconBidAdapter.js [hunk 1]
```diff
@@ -337,7 +337,6 @@ export const spec = {
       'gdpr',
       'gdpr_consent',
       'rp_schain',
-      'rf',
```
Comment (robertrmartinez) [inline modules/rubiconBidAdapter.js]: why is this removed?

Comment (msm0504) [reply modules/rubiconBidAdapter.js]: Duplicate param
